pkg/client/informers/externalversions/generic.go

```diff
@@ -51,11 +51,9 @@ func (f *genericInformer) Lister() cache.GenericLister {
 // ForResource gives generic access to a shared informer of the matching type
 // TODO extend this to unknown resources with a client pool
 func (f *sharedInformerFactory) ForResource(resource schema.GroupVersionResource) (GenericInformer, error) {
-	switch resource {
 	// Group=scaling.k8s.restdev.com, Version=v1alpha1
-	case v1alpha1.SchemeGroupVersion.WithResource("scheduledscalers"):
+	if v1alpha1.SchemeGroupVersion.WithResource("scheduledscalers") == resource {
 		return &genericInformer{resource: resource.GroupResource(), informer: f.Scaling().V1alpha1().ScheduledScalers().Informer()}, nil
```

Comment:
prefer to leave flavor things like this alone to reduce diff with upstream, or make a separate PR in upstream to get it merged first